src/model/mapping/group_views_model.py

```
@@ -0,0 +1,97 @@
+import time
+from pathlib import Path
+
+from langchain.chains.llm import LLMChain
+from pandas import DataFrame, read_excel
+
+from datastep.chains.mapping_nomenclature_view_chain import get_chain
+from infra.env import DATA_FOLDER_PATH
```

Ruff

`infra.env.DATA_FOLDER_PATH` imported but unused

Remove unused import: `infra.env.DATA_FOLDER_PATH`

(F401)